data/json/items/armor/robofac_armor.json

```
@@ -243,6 +243,132 @@
     },
     "flags": [ "DIMENSIONAL_ANCHOR", "PORTAL_PROOF", "STURDY", "OUTER", "WATERPROOF", "VARSIZE", "RAINPROOF", "HOOD", "SOFT" ]
   },
+  {
+    "id": "robofac_head_rig",
+    "repairs_like": "coat_rain",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "Hub 01 modular recon headgear" },
+    "looks_like": "tac_helmet",
+    "description": "Heavy duty hooded headwear for the true frontier explorer, made with robust nylon sheeting reinforced with kevlar fibers. The top consists of a series of plastic plates with hooks and velcro strips, perhaps for attaching additional parts. On the forehead of the assembly is a bulky box visor on a mechanism to quickly pull it down over your eyes, with a cover on the side to insert standard batteries. Activate to lower the visor and use the light amplification functions.",
```

```suggestion
    "description": "Heavy duty hooded headwear for the true frontier explorer, made with robust Nylon sheeting reinforced with Kevlar fibers. The top consists of a series of plastic plates with hooks and velcro strips, perhaps for attaching additional parts. On the forehead of the assembly is a bulky box visor on a mechanism to quickly pull it down over your eyes, with a cover on the side to insert standard batteries. Activate to lower the visor and use the light amplification functions.",
```
*Angry Dupont noises*